drivers/sqlite-driver/src/main/kotlin/com/squareup/sqldelight/sqlite/driver/JdbcSqliteDriver.kt

```diff
@@ -136,6 +140,9 @@ private class SqliteJdbcCursor(
   override fun getDouble(index: Int): Double? {
     return resultSet.getDouble(index + 1).takeUnless { resultSet.wasNull() }
   }
-  override fun close() = resultSet.close()
+  override fun close() {
+    resultSet.close()
+    preparedStatement.close()
```

Comment:
What if we execute this query twice to create two cursors? What effect does closing the prepare statement have?